stingray/bispectrum.py

```diff
@@ -78,6 +79,23 @@ def compute_bispectrum(self, lc):
         bispec = scipy.fftpack.fftshift(bispec) / self.k
         return bispec
 
+    def plot(self):
```

Comment:
sorry i didn't look at #74 . tried to stay coherent with matplotlib ( as many are accustomed to it) hence plot .naming can be discussed ?  @dhuppenkothen